packages/web-components/src/components/va-checkbox/test/va-checkbox.e2e.ts

```diff
@@ -257,4 +257,39 @@ describe('va-checkbox', () => {
     await checkboxLabelEl.click();
     expect(await checkboxEl.getProperty('checked')).toBeTruthy();
   });
+
+  it('sets the data-indeterminate attribute on the input element', async () => {
```

Comment:
I tried to also get a test working that checked if the input element was getting the `.indeterminate` property from being set in JS but I could not get it to work. :/ 

This is kind of a [strange property](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/indeterminate) though so either it's some kind of testing limitation with Stencil or I'm just not writing the test in a way that it can pick up that value.